programs/stake/src/lib.rs

```diff
@@ -23,5 +23,6 @@ pub(crate) fn get_minimum_delegation(_feature_set: &FeatureSet) -> u64 {
     // If/when the minimum delegation amount is changed, the `feature_set` parameter will be used
     // to chose the correct value.  And since the MINIMUM_STAKE_DELEGATION constant cannot be
     // removed, use it here as to not duplicate magic constants.
+    #[allow(deprecated)]
     solana_sdk::stake::MINIMUM_STAKE_DELEGATION
```

Comment:
nit: maybe introduce a crate local constant called `DEFAULT_MIN_STAKE_DELEGATION` to use here instead

Reply:
Heh, I'd like to avoid adding more constants since now we have functions in place to handle if/when the minimum delegation changes. Happy to re-evaluate though; what use-case/problem would a new local constant solve?

Reply:
It's nice to have defaults like this when starting up new clusters. Could also be used in tests. I wish the (now) deprecated constant was named this way from the start :P

Reply:
Is `get_minimum_delegation(&FeatureSet::default())` sufficient?

Reply:
Yeah that's fine! Not a big deal either way